nesta_ds_utils/nesta_io.py

```diff
@@ -0,0 +1,41 @@
+
+from io import BytesIO
+from typing import Union
+import pathlib
```

Comment:
```suggestion
from pathlib import Path
```

^ if this is the only import you can foresee using from `pathlib`

Reply:
Done